### PR TITLE
Feat/logic top page unbonding list

### PR DESCRIPTION
**Pull request by sirius651**
**Pull Request Summary**

Add unbonding list to bind with data. Since not possible to withdraw each dapp currently, use total amount of withdraw

스크린샷 2022-10-25 오전 11 02 28

### CODE REVIEW

File: src/components/dapp-staking-v2/my-staking/UnbondingList.vue [hunk 1]
```diff
@@ -41,33 +42,45 @@
...
   setup(_, { emit }) {
     const { width, screenSize } = useBreakpoints();
     const { myStakeInfos } = useStakerInfo();
-
-    watchEffect(() => {
-      console.log('sss', myStakeInfos.value);
-    });
+    const { unlockingChunks, withdraw } = useUnbonding();
+    const rebond = () => {
+      console.log('rebond');
```
Comment (impelcrypto) [inline src/components/dapp-staking-v2/my-staking/UnbondingList.vue:82]: Do you need this function?

File: src/components/dapp-staking-v2/my-staking/components/modals/ModalRebond.vue [hunk 2]
```diff
@@ -3,33 +3,45 @@
...
-    <div class="box container">
+    <div class="box--container">
       <div>
         <span class="text--title">The amount you wish to rebond</span>
       </div>
-      <div class="text--amount">10,000 ASTR</div>
+      <div class="text--amount"><format-balance :balance="rebondAmount" /></div>
     </div>
-    <astar-button :width="464" :height="52">Confirm</astar-button>
+    <astar-button :width="464" :height="52" @click="confirm">Confirm</astar-button>
```
Comment (impelcrypto) [inline src/components/dapp-staking-v2/my-staking/components/modals/ModalRebond.vue:12]: Let's use i18n

File: src/components/dapp-staking-v2/my-staking/components/modals/ModalWithdraw.vue [hunk 3]
```diff
@@ -1,33 +1,45 @@
 <template>
   <astar-default-modal :width="544" :height="381" :show="show" title="Withdraw" @close="close">
     <div class="text--guide">Your funds are available to withdraw.</div>
```
Comment (impelcrypto) [inline src/components/dapp-staking-v2/my-staking/components/modals/ModalWithdraw.vue:3]: Let's use i18n

File: src/components/dapp-staking-v2/my-staking/items/UnbondingItem.vue [hunk 4]
```diff
@@ -9,31 +9,97 @@
...
+        :height="24"
+        :disabled="item.erasBeforeUnlock !== 0"
+        @click="showWithdrawDialog(item.amount.toString())"
+        >Withdraw</astar-button
+      >
     </div>
+
+    <Teleport to="#app--main">
+      <div :class="'highest-z-index'">
+        <ModalWithdraw
```
Comment (impelcrypto) [inline src/components/dapp-staking-v2/my-staking/items/UnbondingItem.vue:39]: We should use kebab-case.
https://github.com/AstarNetwork/astar-apps/pull/533#discussion_r991776056

File: src/components/dapp-staking-v2/my-staking/items/UnbondingItem.vue [hunk 5]
```diff
@@ -9,31 +9,97 @@
...
     },
   },
+  setup(_, { emit }) {
+    const showModalWithdraw = ref(false);
+    const showModalRebond = ref(false);
+    // MEMO: since not possible to withdraw each chunk currently, use total amount of withdraw
+    const totalAmount = ref('');
+
+    const { withdraw } = useUnbonding();
+    const rebond = () => {
```
Comment (impelcrypto) [inline src/components/dapp-staking-v2/my-staking/items/UnbondingItem.vue:79]: And this.

**Review by impelcrypto — CHANGES_REQUESTED** [review]

**Comment by sirius651** [comment]
Fixed al abovel. let me merge it

**Review by impelcrypto — APPROVED** [review]

**Pull request merged**